test/integration/organisation_test.rb

```diff
@@ -504,13 +504,13 @@ class OrganisationTest < ActionDispatch::IntegrationTest
 
   it "displays breadcrumbs" do
     visit "/government/organisations/prime-ministers-office-10-downing-street"
-    assert page.has_css?(".gem-c-breadcrumbs .gem-c-breadcrumbs--item", text: "Prime Minister's Office, 10 Downing Street")
+    assert page.has_css?(".gem-c-breadcrumbs .gem-c-breadcrumbs__item", text: "Prime Minister's Office, 10 Downing Street")
```

Comment:
Maybe we can one-up this and remove the class name altogether, and test just test that the string appears in the `gem-c-breadcrumbs` element? That way we would have prevented this PR.

Reply:
Fair point, done.